src/core/network.h

```diff
@@ -36,6 +36,11 @@ GIOChannel *g_io_channel_new(int handle);
 /* Returns 1 if IPADDRs are the same. */
 /* Deprecated since it is unused. It will be deleted in a later release. */
 int net_ip_compare(IPADDR *ip1, IPADDR *ip2) G_GNUC_DEPRECATED;
+int g_io_channel_write_block(GIOChannel *channel, void *data, int len);
+int g_io_channel_read_block(GIOChannel *channel, void *data, int len);
+
+/* returns 1 if IPADDRs are the same */
+int net_ip_compare(IPADDR *ip1, IPADDR *ip2);
```

Comment:
Merge conflict messup here? It's already defined above, and deprecated

Reply:
Good catch. Fixed in fixup commit: 3eacf5ed3d0ecb586e82cb1606997f7a6f6e5f61